dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -108,19 +108,19 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
     return (
       <div>
         <form className="container padding-b-bigger" onSubmit={this.handleDeploy}>
+          <div className="col-8">
```

Comment:
AFAIK, col-* should be inside rows - is that right @Angelmmiguel? Does the wordBreak not work if it's in the row?

Reply:
You're right @prydonius. Not wrapping a column inside a row won't break the word-break property but will cause other issues.